### ISSUE TITLE
Allow for a library and database refresh

### ISSUE THREAD

**Issue by Porkepix** [tramhao/termusic#107]
This could be with a shortcut, or with a regular auto-refresh (optional as it could bother some people I guess), or both: it would be nice to have a key to refresh the library and databases or have an auto-resfresh or both so that one doesn't need to reopen the player to consider the new files.

**Comment by tramhao** [comment]
Yes, this is reasonable. For manual refresh, it's easy to add. For auto refresh, it'll need some extra work.

**Comment by hasezoey** [comment]
I know the linked issue is newer, but has some more discussion about how to implement it
